### PR TITLE
MOSIP: 15368 Generated openapi.json for Pre-registration application service.

### PR DESCRIPTION
**Pull request by kameshsr**

### CODE REVIEW

File: pre-registration/pre-registration-application-service/src/test/resources/application.properties [hunk 1]
```diff
@@ -807,4 +808,11 @@ mosip.role.prereg.cancelappointmentpreregid=REGISTRATION_OFFICER,INDIVIDUAL
...
-mosip.role.prereg.getapplicationdetailsappid=REGISTRATION_OFFICER,INDIVIDUAL
\ No newline at end of file
+mosip.role.prereg.getapplicationdetailsappid=REGISTRATION_OFFICER,INDIVIDUAL
+
+mosip.kernel.auth.appids.realm.map={prereg:'preregistration',ida:'mosip',registrationclient:'mosip',regproc:'mosip',partner:'mosip',resident:'mosip',admin:'mosip',crereq:'mosip',creser:'mosip',datsha:'mosip'}
+mosip.id.preregistration.lostuin.delete=mosip.pre-registration.lostuin.delete
+mosip.id.preregistration.updateregistration.delete=mosip.pre-registration.updateregistration.delete
+mosip.preregistration.applications.all.get = mosip.preregistration.applications.all
+preregistration.cookie.contextpath=/preregistration/
+mosip.preregistration.booking.book.id: mosip.pre-registration.booking.book
```
Comment (vishwa-vyom) [inline pre-registration/pre-registration-application-service/src/test/resources/application.properties]: why this property with : and not = ?

File: pre-registration/pre-registration-application-service/src/test/resources/application.properties [hunk 2]
```diff
@@ -440,7 +441,7 @@ mosip.io.prid.url=${kernel.prid.env}/${masterdata.service.version}/${kernel.prid
 # ----------- Document -------------
 
 # masterdata url to get doctype and catcode
-mosip.kernel.masterdata.validdoc.rest.uri=${masterdata.resource.url}/validdocuments/{langcode}
+mosip.kernel.masterdata.validdoc.rest.uri=${masterdata.resource.url}/validdocuments/all?pageSize=50&pageNumber=0
```
Comment (vishwa-vyom) [inline pre-registration/pre-registration-application-service/src/test/resources/application.properties:444]: why is this change done?

File: pre-registration/pre-registration-application-service/src/test/resources/application.properties [hunk 3]
```diff
@@ -84,7 +84,7 @@ auth.server.validate.url=
 global.config.file=application-mz.properties
 pre.reg.config.file=pre-registration-mz.properties
 
-# spring.cloud.config.uri=https://dev.mosip.net/config
+spring.cloud.config.uri=https://dev.mosip.net/config
```
Comment (vishwa-vyom) [inline pre-registration/pre-registration-application-service/src/test/resources/application.properties:87]: If this is not given, is it trying to connect to localhost and failing ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: pre-registration/pre-registration-application-service/src/test/resources/application.properties (hunk 1 above)
Comment (kameshsr) [reply pre-registration/pre-registration-application-service/src/test/resources/application.properties]: Yes i have updated.

File: pre-registration/pre-registration-application-service/src/test/resources/application.properties (hunk 2 above)
Comment (kameshsr) [reply pre-registration/pre-registration-application-service/src/test/resources/application.properties:444]: yes` mosip.kernel.masterdata.validdoc.rest.uri=${masterdata.resource.url}/validdocuments/{langcode}`  
this `{langcode}`  is creating problem. I tried giving langcode=eng and ara also but  it's redirected to this url  https://dev.mosip.net/v1/masterdata/validdocuments/%7langcode    which is bad request. So after changing to mosip.kernel.masterdata.validdoc.rest.uri=${masterdata.resource.url}/validdocuments/all?pageSize=50&pageNumber=0  this is working fine. I have run all the test cases also from root folder all test cases are running fine.

File: pre-registration/pre-registration-application-service/src/test/resources/application.properties (hunk 3 above)
Comment (kameshsr) [reply pre-registration/pre-registration-application-service/src/test/resources/application.properties:87]: Yes, it's trying to fetch application.properties and pre-registration-mz.properties either from local host config or dev config in pre-registration all services they are not using localhost config so i used dev config in localhost:51000/config also it's working.

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**